templates/btax/input_base.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="author" content="">
+    <meta name="description" content="">
+    <meta name="csrf-token" content="{{ csrf_token }}">
+    <!-- <meta name="viewport" content="width=device-width, initial-scale=1"> -->
+
+    {% if GOOGLE_ANALYTICS_PROPERTY_ID %}
+      {% include "ga.html" %}
+    {% endif %}
+
+    <title>Welcome to TaxBrain | TaxBrain</title>
```

Comment:
If we need an equivalent (but new) `input_base.html`, it should not contain references to "TaxBrain", but rather "Cost of Capital Calculator"